index.html

```diff
@@ -207,6 +207,71 @@ <h2 class="headings">Tonics</h2>
 </div>
 
  <div class="card-works">
+  <header class="about-me"> 
+    <h2> About myself</h2>
+  </header>
```

Comment:
- [ ] The `header` tag is used only to wrap the header section of the website. The header section is the topmost part of your app that has the website logo and the navigations, I'll suggest you remove this header tag here as you this is not your website header, :+1: :smile: